### DISCUSSION TITLE
Standalone package override.

### DISCUSSION THREAD

**Discussion by semi710** [nix-community/nixvim#2355] [Q&A] [answered]
I have a standalone Nixvim package, but when I use it in my NixOS config, I want to override some configuration in it.

For example, I want the colorscheme to be generated by my NixOS config. To keep it simpler, let’s say I just want to toggle line numbers.

I was wondering if something like this would be possible:

```nix
inputs.<standalone>.package.default.overrideAttrs {
  config.opts.number = false;
};
```

**Comment by MattSturgeon** [comment, marked as answer]
`overrideAttrs` is for changing the arguments passed to `mkDerivation`.

If you want to change/extend the module config, we provide an `extend` function that takes a module.

```nix
inputs.<standalone>.packages.${system}.default.extend {
  config.opts.number = lib.mkForce false;
};
```

**Reply by semi710** [reply]
Thanks, This is so much powerfull.
